.gitignore

```diff
@@ -36,3 +36,7 @@ env-config.sh
 /calypso-strings.php
 cover.html
 .test.log
+
+.caches/*
+!.caches/.gitkeep
```

Comment:
Do we need to keep the directory in the repository? Will Babel not generate it automatically? Could we run `mkdir -p .cache` in the `test` target?